metaflow/datatools/__init__.py

```diff
@@ -1,2 +1 @@
-
-from .s3 import S3
\ No newline at end of file
+from .s3 import MetaflowS3Exception, S3
```

Comment:
Maybe not related to this PR specifically but `python -m metaflow.datatools.s3op --help` results in a warning 
```
RuntimeWarning: 'metaflow.datatools.s3op' found in sys.modules after import of package 'metaflow.datatools', but prior to execution of 'metaflow.datatools.s3op'; this may result in unpredictable behaviour
```

Reply:
Let me look at that as well. That's weird.

Reply:
It seems to do it before this patch too but checking if there is an easy fix.